python-backend/app/api/mines/mine/resources/mine_map.py

```diff
@@ -54,4 +45,17 @@ def get(self):
         response.last_modified = last_modified
         response.make_conditional(request)
 
-        return response
\ No newline at end of file
+        return response
+
+    def rebuild_map_cache(self):
```

Comment:
Just something to think about here coz I don't have an answer off the top of my head. But it may be worth it to keep the function scope to just get data in the expected format and then storing it into cache in parent/different step?
Not sure, just brain storming here

Reply:
Not sure I understand... I'll chat when you get in.